resources/views/includes/status.blade.php

```diff
@@ -43,17 +43,22 @@ class="text-trwl clearfix">
                             @if (file_exists(public_path('img/' . $status->trainCheckin->HafasTrip->category->value . '.svg')))
                                 <img class="product-icon"
                                      src="{{ asset('img/' . $status->trainCheckin->HafasTrip->category->value . '.svg') }}"
-                                     alt="{{$status->trainCheckin->HafasTrip->category->value}}">
+                                     alt="{{$status->trainCheckin->HafasTrip->category->value}}"
+                                />
                             @else
                                 <i class="fa fa-train d-inline" aria-hidden="true"></i>
                             @endif {{ $status->trainCheckin->HafasTrip->linename }}
                         </span>
                         <span class="ps-2">
-                            <i class="fa fa-route d-inline" aria-hidden="true"></i>&nbsp;
-                            {{number($status->trainCheckin->distance / 1000, 0)}}<small>km</small>
+                            <i class="fa fa-route d-inline" aria-hidden="true"></i>
```

Comment:
Das `&nbsp;` ist wichtig da, damit bei kleinen Displays das Icon und der Wert in einer Zeile bleiben.

```suggestion
                            <i class="fa fa-route d-inline" aria-hidden="true"></i>&nbsp;
```

Reply:
Verstehe dein Argument. Aber so, wie wir das eingebunden haben bringt es nichts, weil wir direkt nach dem non-breaking space noch einen Zeilenumbruch haben. So haben wir dann "zwei Leerzeichen" und damit einen größeren Abstand. Hast du eine Idee, wie man das besser machen könnte?

Reply:
Mhm, dann ists wahrscheinlich wumpe, weils eh schon seit 100 Jahren kaputt ist. Dann resolve ich das jrtzt und merge :)